tools/accuracy_checker/accuracy_checker/adapters/yolo.py

```diff
@@ -1,13 +1,40 @@
-import math
+from collections import namedtuple
 
 import numpy as np
 
 from ..adapters import Adapter
-from ..config import NumberField, StringField, ConfigValidator, ListField, ConfigError
+from ..config import BoolField, NumberField, StringField, ConfigValidator, ListField, ConfigError
 from ..representation import DetectionPrediction
 from ..topology_types import YoloV1Tiny, YoloV2, YoloV2Tiny, YoloV3, YoloV3Tiny
 from ..utils import get_or_parse_value
 
+DetectionBox = namedtuple('DetectionBox', ["x", "y", "w", "h", "confidence", "probabilities"])
+
+
+class YoloOutputProcessor:
+    def __init__(self, coord_correct=None, size_correct=None, conf_correct=None,
+                 prob_correct=None, coord_normalizer=(1, 1), size_normalizer=(1, 1)):
+        self.coord_correct = coord_correct if coord_correct else lambda x: x
+        self.size_correct = size_correct if size_correct else lambda x: np.exp(x)
```

Comment:
no need in lambda, you can directly call numpy function
```suggestion
        self.size_correct = size_correct if size_correct else  np.exp
```